x-pack/plugins/security/server/plugin.ts

```diff
@@ -281,13 +294,28 @@ export class Plugin {
       featureUsage: licensing.featureUsage,
     });
 
+    const clusterClient = core.elasticsearch.client;
     const { watchOnlineStatus$ } = this.elasticsearchService.start();
 
     this.sessionManagementService.start({ online$: watchOnlineStatus$(), taskManager });
-    this.authorizationService.start({
-      features,
-      clusterClient: core.elasticsearch.client,
-      online$: watchOnlineStatus$(),
+    this.authenticationStart = this.authenticationService.start({
+      http: core.http,
+      clusterClient,
+    });
+
+    this.authorizationService.start({ features, clusterClient, online$: watchOnlineStatus$() });
+
+    return Object.freeze<SecurityPluginStart>({
+      authc: {
```

Comment:
**Out of scope**, just asking since I'm here: Do you think we should move the `license` service out of setup and into start as well?

Reply:
Yep, I think it belongs to a `start` contract.